src/main/java/Array9.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Arrays;
+
+public class Array9 {
+
+    public int binarySearch(int[] arr, int x) {
+        int result = -1;
+        int l = 0;
+        int r = arr.length;
+
+        if (arr.length  == 0) return result;
+        if (x < arr[l]) return result;
+        if (x > arr[r - 1]) return result;
+
+            while (r >= l) {
+                int mid = l + (r - l) / 2;
```

Comment:
Không có ý nghĩa lắm về mặt tính toán nhưng cách viết như dưới đây dễ hiểu hơn

```suggestion
                int mid = (l + r) / 2;
```

Reply:
Done